keyform/templates/keyform/home.html

```diff
@@ -70,35 +58,35 @@ <h2 class="title">{% trans 'Keyform Requests' %}</h2>
     {% if is_paginated %}
         <div class='page-bar'>
             <ul>
-                <li class='page'><a href="?page=1&{{ search_data.urlencode }}">First</a></li>
+                <li class='page'><a href="?page=1&{{ search_data.urlencode }}&order={{ order }}">First</a></li>
```

Comment:
> [HTML entities are parsed inside HTML attributes, and a stray & would create an ambiguity. That's why you should always write &amp; instead of just & inside all HTML attributes.](https://stackoverflow.com/a/3705601)